CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## v0.33.7
+
+*August 4, 2020*
+
+### BUG FIXES:
+
+- [go] Build release binary using Go 1.14.4, to avoid halt caused by Go 1.14.1 (https://github.com/golang/go/issues/38223)
```

Comment:
was there a code change for this too anywhere?

Reply:
It was a change to the Docker image used for builds, which lives in a separate repo. Want me to dig it up and include a link to it?

Reply:
nah, that's fine. just thought it would be good to include a PR link if there was one.

Reply:
Tendermint/images. There were no code changes. I just reran the build since we don't specify versions for golang.

Reply:
Looks like the build image change was a noop as well, since we're using the `golang:latest` image.